recipe/sysconfigdata/default/_sysconfigdata_linux.py

```diff
@@ -684,9 +684,24 @@
  'WITH_VALGRIND': 0,
  'X87_DOUBLE_ROUNDING': 0,
  'XMLLIBSUBDIRS': 'xml xml/dom xml/etree xml/parsers xml/sax',
- 'abs_builddir': '/opt/anaconda1anaconda2anaconda3/../work/Python-3.7.3',
- 'abs_srcdir': '/opt/anaconda1anaconda2anaconda3/../work/Python-3.7.3',
+ 'abs_builddir': '/opt/anaconda1anaconda2anaconda3/../work/Python-@PYVERFULL@',
+ 'abs_srcdir': '/opt/anaconda1anaconda2anaconda3/../work/Python-@PYVERFULL@',
  'datarootdir': '/opt/anaconda1anaconda2anaconda3/share',
  'exec_prefix': '/opt/anaconda1anaconda2anaconda3',
  'prefix': '/opt/anaconda1anaconda2anaconda3',
  'srcdir': '.'}
+
+from shutil import which
+from os import environ, devnull
+from subprocess import check_output
+GCC = which('gcc')
+if GCC:
+    with open(devnull, 'w') as err:
+        GCC_VERSION = check_output([GCC, '--version'], stderr=err).decode('utf-8').splitlines()[0].split(' ')[-1]
+    print(GCC_VERSION)
+    for btv in ('BLDSHARED', 'CC', 'CXX', 'LDCXXSHARED',
+                'LDSHARED', 'LINKCC', 'MAINCC'):
+        if btv in build_time_vars:
+            build_time_vars[btv] = build_time_vars[btv]. \
+                replace('-B /opt/anaconda1anaconda2anaconda3/compiler_compat ',
+                        '')
```

Comment:
Should this be added to the other linux platforms?

Reply:
This is used on all linux platforms when our compilers are not in use is it not? Should be?!